### PR TITLE
"order" -John Bercow

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

File: src/main/kotlin/com/openlattice/auditing/AuditInitializationTask.kt [hunk 1]
```diff
@@ -56,7 +57,8 @@ class AuditInitializationTask(
                 UsersAndRolesInitializationTask::class.java,
                 OrganizationsInitializationTask::class.java,
                 Auth0SyncInitializationTask::class.java,
-                EdmSyncInitializerTask::class.java
+                EdmSyncInitializerTask::class.java,
+                IdConstantsReservationTask::class.java
```
Comment (you-leee) [inline src/main/kotlin/com/openlattice/auditing/AuditInitializationTask.kt]: IdConstantsReservationTask is running already after EdmSyncInitializerTask, so I think  EdmSyncInitializerTask can be removed (unless we want list all for clarity)

Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/auditing/AuditInitializationTask.kt]: probably safe to remove but there seems to be some badness happening with lazy evaluation that we should look into at some point so I'd rather leave it for now just to be safe

Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/auditing/AuditInitializationTask.kt]: *evaluation of task order

Comment (you-leee) [reply src/main/kotlin/com/openlattice/auditing/AuditInitializationTask.kt]: Ah, okay! 👍

**Pull request merged**